src/clj/qbits/jet/client/http.clj

```diff
@@ -352,11 +361,15 @@
     (when auth
       (.apply auth request))
 
-    (doseq [[k v] query-string]
-      (if (coll? v)
-        (doseq [v' v]
-          (.param request (name k) (str v')))
-        (.param request (name k) (str v))))
+    (cond
+      (string? query-string)
+      (set-query-string! request query-string)
+      :else
```

Comment:
Nitpick: If there are only two cases here, can please we use `if` instead of `cond`?

Reply:
Done.